python/lib/cloudutils/serviceConfig.py

```diff
@@ -54,7 +54,7 @@ def configration(self):
         except:
             logging.debug(formatExceptionInfo())
             if self.syscfg.env.mode == "Server":
-                raise CloudRuntimeException("Configure %s failed, Please check the /var/log/cloudstack/setupManagement.log for detail"%self.serviceName)
+                raise CloudRuntimeException("Configure %s failed, Please check the /var/log/cloudstack/management/setupManagement.log for detail"%self.serviceName)
```

Comment:
@DaanHoogland You have a valid point, but this is been done throughout all this Python code. Let's fix it later in one go.